schema.sql

```diff
@@ -10,3 +10,5 @@ CREATE TABLE animals (
     neutered            bit,
     weight_kg           decimal           
 );
+
+ALTER TABLE animals ADD species text;
```

Comment:
- [ ] [Optional] Kindly change the data type to varchar, this will allow you to set your text limit as to text which have a fixed max limit